### PR TITLE
Use updated ocserv

### PR DESCRIPTION
**Pull request by aminvakil**
https://github.com/aminvakil/docker-ocserv

https://github.com/TommyLau/docker-ocserv/issues/52

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ services:
       - VPN_IPSEC_PSK=${L2TP_IPSEC_PSK}
       - VPN_USER_CREDENTIAL_LIST=${L2TP_USER_LIST}
   openconnect:
-    image: tommylau/ocserv:latest
+    image: quay.io/aminvakil/ocserv:latest
```
Comment (masihjay-z) [inline docker-compose.yml:24]: why we have to use this ?

File: README.md [hunk 2]
```diff
@@ -60,15 +60,13 @@ All the users opertaions happened while the container is running. If you used a
 If say, you want to create a user named  `tommy`, type the following command
 
 ```
-docker exec -ti openconnect ocpasswd -c /etc/ocserv/ocpasswd -g "Route,All" tommy
+docker exec -ti openconnect ocpasswd -c /etc/ocserv/ocpasswd tommy
```
Comment (masihjay-z) [inline README.md:63]: it helps the users to know about ``-g`` option

File: docker-compose.yml (hunk 1 above)
Comment (aminvakil) [reply docker-compose.yml:24]: `tommylau/ocserv` has not been maintained for quite some time. You can look at the commits on top of tommylau repository (86 so far) and see if it's worth.

File: README.md (hunk 2 above)
Comment (aminvakil) [reply README.md:63]: `tommylau/ocserv` is targeted for users in China, and the default groups are for that, there are not defaults in my `ocserv` docker image.

**Pull request merged**